code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -83,36 +83,38 @@
 
 	if(default_unfasten_wrench(user, I))
 		return
-
 	if(istype(I, /obj/item/reagent_containers) && !(I.item_flags & ABSTRACT) && I.is_open_container())
+		var/obj/item/reagent_containers/B = I
 		. = 1 // no afterattack
 		if(panel_open)
 			to_chat(user, "<span class='warning'>You can't use the [src.name] while its panel is opened!</span>")
 			return
 		if(beaker)
-			to_chat(user, "<span class='warning'>A container is already loaded into [src]!</span>")
-			return
-		if(!user.transferItemToLoc(I, src))
+			eject_beaker(user)
```

Comment:
this is going to result in weirdness due to eject and put in hand before before the user puts in the new beaker; recommend using temporarily transferitemtoloc instead or something.

Reply:
what

Reply:
if eject beaker is called first it will go to the floor instead of hand since the beaker you're trying to put in hasn't actually gone in yet.

Reply:
it is physically impossible to swap a beaker while having a beaker in hand so i ll keep this in for realism